worlds/hades/Options.py

```diff
@@ -0,0 +1,551 @@
+    """Choose the percentage of filler items in the pool that will be to helpers instead. 
+    Helpers give a boost to your max Health or boost the chance of obtaining rare Boons."""
+    display_name = "Filler Helper Percentage"
+    range_start = 0
+    range_end = 100
+    default = 0
+    internal_name = "FillerHelperPercentage"
+
+class MaxHealthHelperPercentage(Range):
+    """Choose the percentage of helper items that will boost your max health."""
+    display_name = "Max Health Helper Percentage"
+    range_start = 0
+    range_end = 100
+    default = 35
+    internal_name = "MaxHealthHelperPercentage"
+
+class InitialMoneyHelperPercentage(Range):
+    """Choose the percentage of helper items that will boost your initial money by 25 each run.
+    This gets capped by the percentage being left from the MaxHealthHelpers. 
+    What percentage remains from this and the MaxHealthHelpers will give you items that boost the 
+    rarity of the boons obtained in runs."""
+    display_name = "Max Health Helper Percentage"
+    range_start = 0
+    range_end = 100
+    default = 35
+    internal_name = "InitialMoneyHelperPercentage"
+
+# -----------------------Settings for Trap -------------------------
+
+class FillerTrapPercentage(Range):
+    """Choose the percentage of filler items in the pool that will be traps instead. 
+    Traps diminish your money or health during a run."""
+    display_name = "Filler Trap Percentage"
+    range_start = 0
+    range_end = 100
+    default = 5
+    internal_name = "FillerTrapPercentage"
+
+# -----------------------Settings for QoL -------------------------
+
+class ReverseOrderExtremeMeasure(Toggle):
+    """When true the order in which extreme meassures applied is reverse 
+    so level 1 is applied to Hades, instead to Meg/The Furies). 
+    For a more balanced experience"""
+    display_name = "ReverseOrderExtremeMeasure"
+    option_true = 1
+    option_false = 0
+    default = 1
+
+class IgnoreGreeceDeaths(Toggle):
+    """If deaths on Greece are ignored for deathlink. Leave off for the memes."""
+    display_name = "IgnoreGreeceDeaths"
+    option_true = 1
+    option_false = 0
+    default = 1
+
+class StoreGiveHints(Toggle):
+    """If seeing an item on the House Contractor's shop/Fated List of Prophecies 
+    should give a hint for it on the multiworld."""
+    display_name = "StoreGiveHints"
+    option_true = 1
+    option_false = 0
+    default = 1
+
+class AutomaticRoomsFinishOnHadesDefeat(Toggle):
+    """If defeating Hades should give all room clears on Room based location mode 
+    or all rooms clears with the equipped weapon on Room weapon based location mode. """
+    display_name = "AutomaticRoomFinishOnHadesDefeat"
+    option_true = 1
+    option_false = 0
+    default = 0
+
+
+# ------------------------------ Building dictionary ------------------------
+
+
+hades_options: typing.Dict[str, type(Option)] = {
+    "initial_weapon": InitialWeapon,
```

Comment:
You need to switch to the new options API. Your PR will not be merged unless this switch is done.
Check the docs, or look at other worlds' options.py for reference.

Reply:
As always, Tunic apworld was really helpful doing this